components/script/dom/htmlelement.rs

```diff
@@ -151,7 +151,11 @@ impl HTMLElementMethods for HTMLElement {
     // https://html.spec.whatwg.org/multipage/#handler-onload
     fn GetOnload(&self) -> Option<Rc<EventHandlerNonNull>> {
         if self.is_body_or_frameset() {
-            window_from_node(self).GetOnload()
+            if document_from_node(self).has_browsing_context() {
+                window_from_node(self).GetOnload()
+            } else {
+                None
```

Comment:
Why is this returning `None` here? Shouldn't it default into returning `self.upcast::<EventTarget>().get_event_handler_common("load")`?